packages/renderer/src/lib/image/LoadImages.svelte

```diff
@@ -104,9 +103,9 @@ async function loadImages(): Promise<void> {
         <div class="flex flex-col grow pl-2">Image Archives</div>
       </div>
     {/if}
-    {#each archivesToLoad as archiveToLoad, index (index)}
+    {#each archivesToLoad as _, index (index)}
       <div class="flex flex-row justify-center w-full py-1">
-        <Input bind:value={archiveToLoad} aria-label="archive path" readonly={true} />
+        <Input bind:value={archivesToLoad[index]} aria-label="archive path" readonly={true} />
```

Comment:
this is weird, we should definitely not do that

Reply:
https://svelte.dev/docs/svelte/compiler-errors#each_item_invalid_assignment

Reply:
omg, this is so not nice, but okay